src/test/java/me/namuhuchutong/step_1/SetCollectionTest.java

```diff
@@ -0,0 +1,64 @@
+package me.namuhuchutong.step_1;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class SetCollectionTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("주어진 numbers의 크기는 3이다.")
+    @Test
+    void given_numbers_size_is_three() {
+        //given
+        int expected = 3;
+
+        //when
+        int result = numbers.size();
+
+        //then
+        assertThat(result).isEqualTo(expected);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```

컬렉션을 쉽게 테스트할 수 있도록 메서드를 제공하고 있어요. 
이번에 사용해 보시면 좋을 것 같습니다.